maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJCMojo.java

```diff
@@ -984,7 +984,29 @@ public void setPlugins(Dependency[] plugins) {
 		this.plugins = plugins;
 	}
 
-	@Component
+    /**
+     * <p>
+     * A list of artifacts to exclude from XJC Plugins resolveTransitively function.
+     * </p>
+     * <p>
+     * If left undefined, then <code>xerces:xercesImpl</code> and <code>xml-apis</code> will
+     * be excluded by default from the classpath resolution.
+     * </p>
+     */
+    @Parameter
+    private String[] artifactExcludes = new String[] {
+        "xerces:xercesImpl", "xml-apis:xml-apis"
```

Comment:
Including default values is a potential breaking change and would require a minor version bump.

Either we:

1. Include default and bump minor version of jaxb-tools

-or-

2. Include the new parameter without a default value and document that these two are the most common

Reply:
Both default aetifacts are known to be classpath messing with the maven plugin when present in the project's classpath and when user configure plugins in maven plugin execution.

The goal here is to provide a configurable way to remove conflicting artifacts with default values that could be overridden by configuration (in order to set to empty or to add other exclusion per choice)

Reply:
What is the config to remove the default exlusions?

Reply:
It's empty tag :)